dietpi/dietpi-software

```diff
@@ -8483,7 +8483,7 @@ server:
 
     log-queries: no
     log-replies: no
-    logfile: /var/log/unbound/unbound.log
+    logfile: /dev/null
```

Comment:
https://linux.die.net/man/5/unbound.conf

It seems to be possible to not set `logfile` at all to skip file logging (reasonable IMO!). Setting it to `/dev/null` theoretically is an overhead size it prepares and actually writes the logs to the blackhole 😉.

Of commenting/skipping the setting does not work, setting it to empty string `""` definitely does according to the manpage: All logs are then written to STDERR which means journalctl in case of systemd unit, exactly what we want 🙂.